lib/createJestRunner.ts

```diff
@@ -78,7 +78,7 @@ export default function createRunner<
       onFailure: OnTestFailure,
       options: TestRunnerOptions,
     ): Promise<void> {
-      const runner: TestRunner = (await import(runPath)).default;
+      const runner: TestRunner = (await import(runPath.toString())).default;
```

Comment:
TS believes this _must_ be a string, which is wrong. But it's also fine to just stringify it (since it's transpiled to `require` it's probably a good idea)